ReactAndroid/src/main/java/com/facebook/react/modules/clipboard/ClipboardModule.java

```diff
@@ -43,28 +44,30 @@ public String getName() {
   }
 
   private ClipboardManager getClipboardService() {
-    ReactApplicationContext reactContext = getReactApplicationContext(); 
-    return (ClipboardManager) reactContext.getSystemService(reactContext.CLIPBOARD_SERVICE);
+    return (ClipboardManager) getReactApplicationContext().getSystemService(getReactApplicationContext().CLIPBOARD_SERVICE);
+  }
+
+  private String getStringInternal(){
```

Comment:
Nor sure about the naming :)

Probably just inline this in the "getString" method as we don't use it anywhere else :D